src/Util/Api4ArgParser.php

```diff
@@ -4,6 +4,23 @@
 
 class Api4ArgParser extends AbstractPlusParser {
 
+  /**
+   * @var string
+   */
+  private $operators;
+
+  public function __construct() {
+    if (is_callable(['\Civi\Api4\Utils\CoreUtil', 'getOperators'])) {
+      // 5.30+
+      $this->operators = implode('|', ArrayUtil::map(\Civi\Api4\Utils\CoreUtil::getOperators(), function($op) {
+        return preg_quote($op, '/');
+      }));
+    }
+    else {
+      $this->operators = '\<=|\>=|=|!=|\<|\>|IS NULL|IS NOT NULL|IS EMPTY|IS NOT EMPTY|LIKE|NOT LIKE|IN|NOT IN|CONTAINS|NOT CONTAINS|REGEXP|NOT REGEXP|REGEXP BINARY|NOT REGEXP BINARY';
```

Comment:
I mean, if you really wanna support < 5.30, then this list is a bit long. Those weren't all available at the time.

Reply:
Well... *shrug*. I just don't want the parser to crash where it was non-crashing before.

As to which operators work on which exact versions -- that's not really the concern of `cv`. The goal is to pass-through data to `civicrm_api4()` -- then `civicrm_api4()` can figure out whether it likes the request. (*Same as with entity-names and action-names.*)